cypress/fixtures/commonComponents/loader.json

```diff
@@ -1,4 +1,7 @@
 {
+  "sizeDefault": {
+
```

Comment:
Should this object have something in it?

Reply:
nope :) it should apply the default option for `size="medium"`;